Jenkinsfile

```diff
@@ -138,28 +149,40 @@ pipeline {
       }
     }
 
-    stage('Update Github'){
+    // open a Bump PR against dcos/dcos
+    stage('Release Version'){
       when {
-        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+        expression {
+          master_branches.contains(BRANCH_NAME) && params.CREATE_RELEASE == true
+        }
       }
 
       steps {
         withCredentials([
-          usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER')
+            string(credentialsId: '3f0dbb48-de33-431f-b91c-2366d2f0e1cf',variable: 'AWS_ACCESS_KEY_ID'),
+            string(credentialsId: 'f585ec9a-3c38-4f67-8bdb-79e5d4761937',variable: 'AWS_SECRET_ACCESS_KEY'),
+            usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER')
         ]) {
-          sh "GIT_PASSWORD=${GIT_PASSWORD} GIT_USER=${GIT_USER} ./scripts/ci/update-github"
+          sh "git config --global user.email $GIT_USER@users.noreply.github.com"
+          sh "git config --global user.name 'MesosphereCI Robot'"
+          sh "git config credential.helper 'cache --timeout=300'"
+
+          sh "CREATE_RELEASE=1 PUSH_RELEASE=1 ./scripts/ci/release-version"
         }
       }
     }
 
     // trigger the other job to update the upstream reference
-    stage ('Trigger Enterprise Update') {
+    stage ('Run Enterprise Job') {
       when {
-        expression { env.BRANCH_NAME == 'master' || env.BRANCH_NAME =~ SEMVER_REGEX }
+        expression { master_branches.contains(BRANCH_NAME) }
```

Comment:
this will call Enterprise Pipeline and pass the parameter to that job, so there will only be a release if the parameter is checked (Enterprise Pipeline is TODO currently). If not, enterprise will only upload latest, not release is created.

Reply:
Yes makes sense.